firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalStoreTestCase.java

```diff
@@ -152,23 +151,25 @@ private void udpateViews(int targetId, boolean fromCache) {
     notifyLocalViewChanges(viewChanges(targetId, fromCache, asList(), asList()));
   }
 
-  private void acknowledgeMutation(long documentVersion, @Nullable Object transformResult) {
+  private void acknowledgeMutationWithTransform(long documentVersion, Object... transformResult) {
     MutationBatch batch = batches.remove(0);
     SnapshotVersion version = version(documentVersion);
-    MutationResult mutationResult =
-        new MutationResult(
-            version,
-            transformResult != null
-                ? Collections.singletonList(TestUtil.wrap(transformResult))
-                : Collections.emptyList());
+    List<MutationResult> mutationResults =
+        Collections.singletonList(new MutationResult(version, emptyList()));
+    if (transformResult != null) {
```

Comment:
This array should never be null. You can also use Java8 streams to simplify this logic.

Reply:
It is null when `acknowledgeMutation` is called.

Reply:
You should call it with no arguments instead of `null`.

Reply:
Bump :)

Reply:
Fixed.